Docs/index.md

```diff
@@ -0,0 +1,46 @@
+# Welcome to the Santa Docs
+
+Santa is a binary whitelisting/blacklisting system for macOS. Here you will find the documentation for understanding how Santa works, how to deploy it and how to contribute.
+
+#### Introduction
+
+The following documents give an overview on how Santa accomplishes binary whitelisting/blacklisting at the enterprise scale.
```

Comment:
The following documents give an overview of how Santa accomplishes binary whitelisting/blacklisting at the enterprise scale.